script.artwork.downloader/default.py

```diff
@@ -681,6 +681,7 @@ def _gui_mode(self, currentmedia):
                    background = setting['background'])
         # Look for argument matching artwork types
         for item in sys.argv:
+            #log('HHH    ' + item) #burekas debug
```

Comment:
Why add a commented line?

Reply:
This remained there due to a small debug that I was try to investigate about specific case (In TV Shows, when you press on DiscArt it's still looking for 'banners' as arg) . But I still haven't found the reason why. Not so critical right now.
I will remove it. Thanks.